src/MizarData/include/MizarData/BaselineAndComparison.h

```diff
@@ -16,13 +18,14 @@
 #include "MizarData/NonWrappingAddition.h"
 #include "MizarData/SampledFunctionId.h"
 #include "MizarData/SamplingWithFrameTrackComparisonReport.h"
+#include "Statistics/MultiplicityCorrection.h"
 
 namespace orbit_mizar_data {
 
 // The class owns the data from two capture files via owning two instances of
 // `PairedData`. Also owns the map from sampled function ids to the
 // corresponding function names.
-template <typename PairedData, typename FunctionTimeComparator>
+template <typename PairedData, typename FunctionTimeComparator, auto MultiplicityCorrection>
```

Comment:
Why is this `auto`?

Reply:
Because this is a non-type template argument -- we will pass a function there. Writing
`template <auto T> struct S{...};` 
is close to (or even the same as?) 
`template <typename Type, Type Value> S {...}`

Reply:
Okay, thanks.